open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -73,8 +73,8 @@ igc:
 image:
   ranger:
     repository: planetf1/ranger-admin-egeriavdc
```

Comment:
These are all still hard-coded to your personal Docker repo...  Is there not some way in Helm to have a single parameter for repository, and then use `{{ repo }}` (or similar) in each of these to repeat it (so there's only one value to overload)?

Reply:
yes undersood -- actually the default is encoded for sure, but all these values can be overriden in helm already .

This issue was concerned with the maven build process only... and minimal changes to allow things to still work (hence why the tags got updated). refactoring of the helm charts will be in another change. Baby steps. small increments. There are so many moving parts. My plan is to simplify the override by allowing a user to specify the repository base name just once rather than multiple times, and probably similar for the tags

I'd propose to leave a default in place -- the images indicated are on docker hub so available to all, which means they would work for a chart consumer with no changes. At some point I hope we can get images under the 'odpi' account, at which point the default will change away from my account. But we don't have that capability yet,.